flask_common/utils.py

```diff
@@ -719,7 +719,10 @@ def get_ignored_collections(self):
 
 
     def _get_queries(self):
-        ignore_query = { "ns": {"$nin": self.get_ignored_collections()}, "op": { "$ne": "killcursors" } }
+        ignore_query = { "$or": [
+            { "ns": {"$nin": self.get_ignored_collections()}, "op": { "$ne": "killcursors" } },
+            { "ns": "{0}.$cmd".format(self.db.name), "command.findAndModify": { "$exists": True } },
```

Comment:
Why do we explicitly need to include these? Where are they removed?

If the findAndModify ops need to be explicitly included, It's pretty unintuitive to do so in an `ignore_query`.